config/config.go

```diff
@@ -0,0 +1,81 @@
+package config
+
+import (
+	"os"
+	"path"
+	"strings"
+
+	"github.com/daedaleanai/dbt/log"
+	"github.com/daedaleanai/dbt/util"
+	"gopkg.in/yaml.v2"
+)
+
+type Config struct {
+	Mirror string
+}
+
+var environment map[string]string
+var config *Config
+
+const configFileName string = "config.yaml"
+
+func init() {
+	environment = make(map[string]string)
+	for _, v := range os.Environ() {
+		parts := strings.Split(v, "=")
+		if len(parts) == 2 {
+			key := parts[0]
+			value := parts[1]
+			environment[key] = value
+		}
+	}
+}
+
+func getDbtConfigDir() string {
+	dbtConfigDir, ok := environment["DBT_CONFIG_DIR"]
+	if ok {
+		return dbtConfigDir
+	}
+
+	xdgConfigDir, ok := environment["XDG_CONFIG_DIR"]
```

Comment:
what does XDG stand for?

Reply:
It's a standard in linux that specifies the location of certain configuration and data files. The environment variables can be defined by the user to change where applications look for these configuration files:
`https://wiki.archlinux.org/title/XDG_Base_Directory`

But reading it again, I need to change it, because it should be `XDG_CONFIG_HOME`.